tools/csv-generator/csv-generator.go

```diff
@@ -166,6 +166,9 @@ Performance Addon Operator provides the ability to enable advanced node performa
 		operatorCSV.Annotations["olm.skipRange"] = *skipRange
 	}
 
+	operatorCSV.Annotations["description"] = "Operator to optimize OpenShift clusters for applications sensitive to CPU and network latency."
+	operatorCSV.Annotations["repository"] = "https://github.com/operator-kni/performance-addon-operators"
```

Comment:
we have an option to inject annotations, this change is unneeded

Reply:
How? After this PR everything is generated, manual changes to csv won't do. If you depend on env variables, the CI and local builds  should have the same env variables set

Reply:
I meant that `annotations-from` option of this tool. And the tool should keep working after this PR merged. But we can fix afterwards I guess.

Reply:
See line 100 above

Reply:
Right, I actually missed that, anyway - this is a "fix description" - and we also may want to move at removing the generator altogether.

Reply:
Not in this PR please :) I think this tool still have a strong use case. Or to put it differently, we need to figure out a well defined alternative/flow.